src/site/content/en/design-system/render-component-test.njk

```diff
@@ -0,0 +1,24 @@
+---
+permalink: '/design-system/render-component-test/index.html'
+title: 'Test'
+button: 
+  text: 'Override data'
+aside:
+  content: 'Some **markdown** in here'
+---
+
+{% extends "design-system.njk" %}
+
+{% block inner %}
+
+{{ design.components.render('button') | safe }}
+{{ design.components.render('button-primary') | safe }}
+{{ design.components.render('button-primary', button) | safe }}
+
+{{ design.components.render('aside') | safe }}
+{{ design.components.render('aside-warning', aside) | safe }}
+
+{# Should throw a console error #}
```

Comment:
Throw implies the site fails to build, and clearly it builds fine. I'm in two minds, I actually wouldn't mind this to crash, so in this case could you just comment this out and say "this would have crashed"?

Reply:
I'll just delete this page once this PR is approved anyway. It's a temporary test